### DISCUSSION TITLE
disable the rabbit but keep it things it will be very useful for my type of pack

### DISCUSSION THREAD

**Discussion by KibaTheBarbarian** [Roadhog360/Et-Futurum-Requiem#512] [Ideas]
Can you add a config to disable rabbits but keep its things like the drops and the stuff you craft with the drops

**Comment by Roadhog360** [comment]
Aren't there mods that can block spawning of certain mobs?

**Comment by KibaTheBarbarian** [comment]
There is I'm just wondering if it was easy to do on your side just in case people have other rabbit mobs in their mod pack from Twilight forest or some other thing sorry about if this was wasting your time

**Reply by Roadhog360** [reply]
Will think about it
